src/helpers/arbeidssituasjon/aktivitetvalidering.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  EStilling,
+  IArbeidsgiver,
+} from '../../models/steg/aktivitet/arbeidsgiver';
+import {
+  EAktivitet,
+  IAksjeselskap,
+  IAktivitet,
+} from '../../models/steg/aktivitet/aktivitet';
+import { harValgtSvar } from '../../utils/spørsmålogsvar';
+import {
+  EStudieandel,
+  IUnderUtdanning,
+} from '../../models/steg/aktivitet/utdanning';
+
+export const erSisteArbeidsgiverFerdigUtfylt = (
+  arbeidsforhold: IArbeidsgiver[]
+) => {
+  return arbeidsforhold?.some(
```

Comment:
Hvorfor holder det med `some` her? Må ikke alle være utfylt? Isåfall; `every` ?

Reply:
Jo, alle felter skal være utfylt :) 
Her sjekker jeg bare den siste arbeidsgiveren siden spørsmålene må besvares før man kan gå videre, men når jeg tenker over det det stopper ikke søkeren fra å gå tilbake å "fjerne" svaret sitt.. Jeg endrer til` every` og sjekker :)

Reply:
Mm, ser nå at `every` ikke løser det at man kan gå tilbake og fjerne svaret sitt.. Men det gjør ikke `some` heller. Må kanskje utvide sjekken til å i stedet sjekke kun de siste feltene, så må man sjekke alle feltene før neste knappen kan vises. 🤔 Skal se nærmere på det.